tock/tock/templates/hours/dashboard.html

```diff
@@ -27,93 +27,108 @@
 <i>{{ fytd_start_date }} to {{ rp_selected.end_date }}</i>
 <h4>Hours</h4>
 <table>
-  <tr>
-    <th></th>
-    <th>FYTD Target</th>
-    <th>FYTD Performance</th>
-    <th>Variance</th>
-  </tr>
-  <tr>
-    <td>Cost Recovery</td>
-    <td>{{ hours_required_cr_fytd }}</td>
-    <td>{{ hours_billed_fytd }}</td>
-    <td>{{ variance_cr_ytd }} ({{ p_variance_cr_fytd }})</td>
-  </tr>
-  <tr>
-    <td>Financial Plan</td>
-    <td>{{ hours_required_plan_fytd }}</td>
-    <td>{{ hours_billed_fytd }}</td>
-    <td>{{ variance_plan_fytd }} ({{ p_variance_plan_fytd }})</td>
-  </tr>
+  <thead>
+    <tr>
+      <th scope="col"></th>
+      <th scope="col">FYTD Target</th>
+      <th scope="col">FYTD Performance</th>
+      <th scope="col">Variance</th>
+    </tr>
+  <tbody>
+    <tr>
+      <th scope="row">Cost Recovery</th>
+      <td>{{ hours_required_cr_fytd }}</td>
+      <td>{{ hours_billed_fytd }}</td>
+      <td>{{ variance_cr_ytd }} ({{ p_variance_cr_fytd }})</td>
+    </tr>
+    <tr>
+      <th scope="row">Financial Plan</th>
+      <td>{{ hours_required_plan_fytd }}</td>
+      <td>{{ hours_billed_fytd }}</td>
+      <td>{{ variance_plan_fytd }} ({{ p_variance_plan_fytd }})</td>
+    </tr>
+  </tbody>
 </table>
 <br>
 <h4>Revenue</h4>
 <table>
+  <thead>
+    <tr>
+      <th></th>
+      <th scope="col">FYTD Target</th>
+      <th scope="col">FYTD Performance</th>
+      <th scope="col">Variance</th>
+    </tr>
+  </thead>
+  <tbody>
   <tr>
```

Comment:
The whole code block below needs to be indented, not just that line. Sorry if that wasn't clear.